examples/pion-to-pion-trickle/answer/main.go

```diff
@@ -99,12 +110,14 @@ func main() {
 		} else if closeErr := resp.Body.Close(); closeErr != nil {
 			panic(closeErr)
 		}
+		answerwg.Add(1) //We have now sent the answer and can send candidates to the peer
```

Comment:
Shouldn't this be `answerwg.Done()`?

Reply:
Hey @arlolra

Thanks for the review, we ended up dropping the WaitGroup because it didn't work in WASM https://github.com/pion/webrtc/blob/master/examples/pion-to-pion-trickle/answer/main.go

Hopefully this issue wasn't ported over!

Reply:
Oh thanks for catching that arlo!

@Sean-Der does this fix make it work in WASM?

Reply:
@cohosh 

Yes! It now compiles for WASM with this change. I don't actually use the WASM implementation myself so don't know all the deep details sorry :/